app/models/concerns/locatable.rb

```diff
@@ -0,0 +1,11 @@
+module Locatable
+  extend ActiveSupport::Concern
+
+  included do
+    acts_as_mappable lat_column_name: :latitude, lng_column_name: :longitude
```

Comment:
This might not be necessary since the `Location` model now has lat/lng fields on it. Will investigate…